src/AlertFlags/AlertFlags.js

```diff
@@ -1,7 +1,17 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
 import './AlertFlags.scss'
 
+const Flag = ({ children, ...props }) => (
+  <Fragment>
```

Comment:
Could this be AlertFlag or something? Just thinking about confusing it with the `Flag` library component we use for feature toggles. Not a blocker as doesn't break anything, just makes searching for things a bit harder.

Reply:
Yeah good shout!